lib/process_sonos_command.js

```diff
@@ -29,9 +29,12 @@ export default async function process_sonos_command(received_text) {
   } else if (received_text_lower.startsWith('spotify:')) {
     service_type = 'spotify';
     sonos_instruction = 'spotify/now/' + received_text;
-  } else if (received_text_lower.startsWith('tunein:')) {
+  } else if (received_text_lower.startsWith('tunein/')) {
     service_type = 'tunein';
-    sonos_instruction = 'tunein/now/' + received_text;
+    sonos_instruction = received_text;
+  } else if (received_text_lower.startsWith('favorite/')) {
```

Comment:
```suggestion
  } else if (received_text_lower.startsWith('favorite:')) {
```

Again for consistency let's program cards with colon delimiters. OK to also accept "slashes" if you want to add that.